GOVERNANCE.md

```diff
@@ -0,0 +1,170 @@
+
+Implementers are those who have implemented the CDS Hooks API. Such implementations can be done locally, in a test environment, or in a production system.
+
+### The Community
+
+The community refers to the collective set of individuals and parties contributing to, implementing, or following the CDS Hooks project.
+
+## Accepting New Committers and PMC Members
+
+Any existing committer may nominate a contributor as a new committer on the project. The nominee should have a demonstrated history of contributions and commitment. Contributions can come in many forms such as code, issues, documentation, and community engagement. A nominee must receive a unanimous vote from the PMC. Upon being accepted, the PMC Chair will grant the nominee write access to the CDS Hooks repositories and announce the new committer to the mailing list.
+
+Any PMC member may nominate a current committer to the join the PMC. The nominee should have a demonstrated history leadership in support of the community, and must receive a unanimous vote from the PMC.
+
+## Contributions
+
+Anyone is welcome to contribute to the project. All contributions are public and associated to the individual(s) submitting the change.
+
+### Master branch
+
+The master branch must be kept in a releasable state.
+
+Trivial changes may be committed directly to a repository. Changes deemed trivial are things like small documentation typos, ecosystem changes (eg, CI configuration, linting rules), etc. Obviously, *trivial* is subjective and anyone is welcome to comment directly on such changes or open an issue regarding them.
+
+All other changes must be reviewed by the committers. This includes contributions from the community (contributors) as well as the current committers. We use [Github pull requests](https://help.github.com/articles/about-pull-requests/) and [Github pull request reviews](https://help.github.com/articles/about-pull-request-reviews/) to manage reviewing and merging the proposed changes.
+
+For small changes, at least one committer must approve the change. If a committer was the author of the change being reviewed, the approval must come from a different committer. For large, significant, or breaking changes, the committers must reach consensus on the change. Like *trivial*, the notion of *small* and *large* are subjective terms. However, a breaking change should be clear and unambiguous. Note that the size of the contribution does not determine whether the change requires just one approval or consensus. Rather, the impact of the change reflects the type of review required. For instance, the addition of a single optional field to the API would be considered a significant change and would warrant consensus amongst the committers.
+
+API changes are scrutinized far more closely than that of documentation or related code projects (like sample projects, test harnesses, etc). As such, API changes are always considered significant and require consensus from the committers.
```

Comment:
We haven't talked much of the FHIRcast API. Should this be *specification* instead?

Reply:
solid point, @lbergnehr ! Changed. 

Also, awesome review! 👍